PowerShellTools/Intellisense/IntelliSenseManager.cs

```diff
@@ -119,7 +119,14 @@ public int Exec(ref Guid pguidCmdGroup, uint nCmdId, uint nCmdexecopt, IntPtr pv
                 return VSConstants.S_OK;
             }
 
-            //pass along the command so the char is added to the buffer 
+            // Check the char at caret before pass along the command
+            // If command is backspace and completion session is active, then we need to see if the char to be deleted is an IntelliSense triggering char
+            // If yes, then after deleting the char, we also dismiss the completion session
+            // Otherwise, just filter the completion lists
+            ITrackingPoint caretCharPosition = _textView.TextSnapshot.CreateTrackingPoint(_textView.Caret.Position.BufferPosition.Position - 1, PointTrackingMode.Positive);

```

Comment:
Caret is always +1 compared to the char pos. e.g., at the very beginning of the file, caret pos == 1.